bin/zField

```diff
@@ -150,8 +156,37 @@ if __name__ == '__main__':
             dMapDict = clusters.makeDensityMap(RADeg, decDeg, galaxyCatalog, z, dz, rMaxMpc = 1.5)
             astImages.saveFITS(args.outDir+os.path.sep+"densityMap_z%s_%.3f_%.3f.fits" % (z, RADeg, decDeg),
                                dMapDict['map'], dMapDict['wcs'])
-    
+            
+    # Estimate errors for density map, CS and A
+    if ErrMap == True:
+        print("Starting MC 1 of %..."%(MC))
```

Comment:
Pretty sure I wrote this the other day in a more general comment. This will give an error (you would want `%d` not `%..`.). Just delete this line and write something like:

`print(">>> Estimating galaxy density error map ...")`

If you run this code as is, you will get:

`ValueError: unsupported format character '.' (0x2e) at index 19`

Don't you test your code?